### PR TITLE
Improvements to tenant language functionality

### PR DESCRIPTION
**Pull request by ianlchapman**
Fixed bug where tenant language files were not being updated if global language files exist

Added functionality for tenant language files to extend and override rather than replace global language files.

### CODE REVIEW

File: src/Translations/MultiFileLoader.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    /**
+     * Create a new file loader instance.
+     *
+     * @param  \Illuminate\Filesystem\Filesystem  $files
+     * @param  string  $path
+     * @return void
+     */
+    public function __construct(Filesystem $files, $path)
+    {
+        parent::__construct($files, $path);
```
Comment (luceos) [inline src/Translations/MultiFileLoader.php:37]: There's no need to define the construct method if all it does is proxy to the parent.

File: src/Translations/MultiFileLoader.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+     * @param  string  $locale
+     * @param  string  $group
+     * @param  string  $namespace
+     * @return array
+     */
+    public function load($locale, $group, $namespace = null)
+    {
+        $results = [];
+        foreach ($this->fileLoaders as $loader) {
+            $results = array_merge($results, $loader->load($locale, $group, $namespace));
```
Comment (luceos) [inline src/Translations/MultiFileLoader.php:61]: I think you should create a test that confirms the precedence of the existing global translation over the tenant translation. From what I read here, any subsequent loader will override the parent.

**Review by luceos — CHANGES_REQUESTED** [review]
I've added a few comments and asked @Thijmen to do a review too.

Sorry to be a bother, but this is a pretty important part of the code base ;) Well done btw.

File: src/Translations/MultiFileLoader.php (hunk 2 above)
Comment (Thijmen) [reply src/Translations/MultiFileLoader.php:61]: I agree, I would like to see a test for this.

**Comment by luceos** [comment]
@ianlchapman would you have time on your hands for that last hurdle? Please let me know if you need assistance.

**Comment by Thijmen** [comment]
@ianlchapman Can you write some tests as reviewed above?

**Comment by bkintanar** [comment]
@ianlchapman let us know if you can write some tests on this so we could review.

**Comment by luceos** [comment]
@Thijmen @bkintanar let's do this ourselves. If either of you is up for it, assign the pr to yourself. Otherwise I'll pick it up within a week.

**Pull request merged**